WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -22,10 +22,28 @@ extension PostEditor where Self: UIViewController {
 
     func displayHistory() {
         let revisionsViewController = RevisionsTableViewController(post: post) { [weak self] revision in
-            if let post = self?.post.update(from: revision) {
-                DispatchQueue.main.async {
-                    self?.post = post
+            guard let self = self else {
```

Comment:
I think unwrapping `self` here is not necessary and you can `guard let post = self.post.update(from: revision)` instead.